terragrunt/aws/csp_violation_report_service/lambda.tf

```diff
@@ -0,0 +1,56 @@
+module "purge_csp_reports_lambda" {
+  source                 = "github.com/cds-snc/terraform-modules?ref=v3.0.5//lambda"
+  name                   = "purge_csp_reports"
+  billing_tag_value      = var.billing_tag_value
+  ecr_arn                = aws_ecr_repository.purge_csp_reports.arn
+  enable_lambda_insights = true
+  image_uri              = "${aws_ecr_repository.purge_csp_reports.repository_url}:latest"
+  memory                 = 512
+  timeout                = 300
+
+  vpc = {
+    security_group_ids = [aws_security_group.csp_reports.id, module.csp_reports_db.proxy_security_group_id]
+    subnet_ids         = var.vpc_private_subnet_ids
+  }
+
+  environment_variables = {
+    DB_HOST                 = aws_ssm_parameter.db_host.value
+    DB_USERNAME             = aws_ssm_parameter.db_username.value
+    DB_DATABASE             = aws_ssm_parameter.db_database.value
+    DB_PASSWORD             = aws_ssm_parameter.db_password.value
```

Comment:
Could these be loaded on cold start directly from SSM rather than as `environment_variables`?  I'm permanently scared by Notify because of this...